draft-ietf-tls-tls13.md

```diff
@@ -2183,8 +2183,10 @@ hidden by TCP.
 
 A small allowance for errors in clocks and variations in measurements
 is advisable.  However, any allowance also increases the opportunity
-for replay.  In this case, it is better to reject early data than to
-risk greater exposure to replay attacks.
+for replay.  In this case, it is better to reject early data and fall back
+to a full 1-RTT handshake than to risk greater exposure to replay attacks,
+so in common network topologies, a small allowance on the order of ten
+seconds is reasonable.
```

Comment:
it would also be useful to mention that they should consider and lower and upper bound separately. This will help implementations provide the right api.